jbmc/src/Makefile

```diff
@@ -40,11 +41,33 @@ generated_files: $(patsubst %, %_generated_files, $(DIRS))
 # cleaning
 
 .PHONY: clean
-clean: $(patsubst %, %_clean, $(DIRS)) cprover_clean
+clean: $(patsubst %, %_clean, $(DIRS)) cprover_clean dist_clean
 
 $(patsubst %, %_clean, $(DIRS)):
 	$(MAKE) $(MAKEARGS) -C $(patsubst %_clean, %, $@) clean ; \
 
 .PHONY: cprover_clean
 cprover_clean:
 	$(MAKE) $(MAKEARGS) -C $(CPROVER_DIR)/src clean
+
+.PHONY: dist_clean
+dist_clean:
+	rm -rf $(ROOT)dist
+
+# extended JBMC models download, for your convenience
+java-models-library-download:
+	@echo "Downloading java models library"
+	@curl -L https://github.com/diffblue/java-models-library/archive/master.zip > java-models-library.zip
+	@unzip java-models-library.zip
+	@rm java-models-library.zip
+	@cp -r java-models-library-master/src java_bytecode/library
+	@rm -r java-models-library-master
+
+.PHONY: dist
+dist: java-models-library-download all
+	@mkdir --p $(ROOT)dist/lib
+	@cp java_bytecode/library/core-models.jar $(ROOT)dist/lib
+	@mkdir --p $(ROOT)dist/bin
+	@cp jbmc/jbmc $(ROOT)dist/bin
+	@cp janalyzer/janalyzer $(ROOT)dist/bin
```

Comment:
I would definitively like to see what `make dist` is doing when I type it, better to remove the `@`.

Reply:
done